includes/api/podcast.php

```diff
@@ -33,6 +33,12 @@ public function register_routes()
             array_push( $categories_enum, $val );
         }
 
+        $locales = \Podlove\Locale\locales();
+        $locales_enum = [];
+        foreach ($locales as $key => $val) {
+            array_push( $locales_enum, $key );
+        }
+
```

Comment:
Ich glaube das ist das gleiche:

```suggestion
        $locales = \Podlove\Locale\locales();
        $locales_enum = array_keys($locales);
```